int8 id
int8 mode
float32 tau
float32 x
float32 dx